server/broadlistening/pipeline/steps/hierarchical_aggregation.py

```
@@ -240,5 +356,38 @@ def _build_property_map(
+                        value = str(value)
+                    except:
+                        value = None
+
+            # Make sure arg_id is string
+            str_arg_id = str(arg_id)
+            property_map[prop][str_arg_id] = value
+
     return property_map
+
+
+def json_serialize_numpy(data):
```

Ruff (0.11.9)

Redefinition of unused `json_serialize_numpy` from line 15

(F811)